package.json

```diff
@@ -1,63 +1,49 @@
 {
 	"name": "@monogram/eslint-config",
-	"version": "0.1.2",
+	"version": "0.2.0",
 	"description": "ESLint configs used at Monogram",
 	"keywords": [
 		"eslint",
-		"eslint-config-airbnb",
-		"eslint-config-next"
+		"eslintconfig",
+		"eslint-config"
 	],
 	"license": "MIT",
 	"scripts": {
-		"update-local": "pnpm build && pnpm install-locally",
-		"install-locally": "npm install -g",
-		"prepublish": "pnpm build && pnpm release",
-		"build": "tsc --project tsconfig.json",
-		"prestart": "pnpm build",
-		"start": "ts-node src/index.ts",
-		"release": "standard-version",
-		"postinstall": "pnpm build"
```

Comment:
why remove the existing scripts?

Reply:
I kept `install-locally` (also serves as `update-local`), `build`,  and `prepublish`.

I removed `prestart` because the `start` one doesn't even use the build file, so it doesn't make sense.

I guess I can add back `start` for local testing of the npx script.

I removed `release` because "standard-version" package is deprecated.

I removed `postinstall` because I believe it's unnecessary.